frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -61,52 +69,51 @@ describe('Kubevirt create VM using wizard', () => {
     removeLeakedResources(leakedResources);
   });
 
-  provisionConfigs.forEach((provisionConfig, configName) => {
+  for (const [method, vm] of Object.entries(vmPresets)) {
     const specTimeout =
-      configName === ProvisionConfigName.DISK ? CLONE_VM_TIMEOUT_SECS : VM_BOOTUP_TIMEOUT_SECS;
+      method === ProvisionSource.DISK ? CLONE_VM_TIMEOUT_SECS : VM_BOOTUP_TIMEOUT_SECS;
+
     it(
-      `${VMTestCaseIDs[configName]} Create VM using ${configName}.`,
+      `${VMTestCaseIDs[method]} Create VM using ${method}.`,
       async () => {
-        const vm = await wizard.createVirtualMachine(
-          vmConfig(configName.toLowerCase(), testName, provisionConfig),
-        );
         await withResource(leakedResources, vm.asResource(), async () => {
+          await vm.create();
           await vm.navigateToDetail();
         });
       },
       specTimeout,
     );
-  });
+  }
 
   it('ID(CNV-3657) Creates VM with CD ROM added in Wizard', async () => {
-    const vmName = 'vm-with-cdrom';
-    const provisionConfig = provisionConfigs.get(ProvisionConfigName.CONTAINER);
-    provisionConfig.CDRoms = [cdGuestTools];
-    const vm = await wizard.createVirtualMachine(
-      vmConfig(vmName, testName, provisionConfig, basicVMConfig, false),
-    );
+    const vm = new VMBuilder(basicVM)
+      .setProvisionSource(provisionSources.Container)
+      .setDisks([cdGuestTools])
+      .generateName()
+      .build();
+
     await withResource(leakedResources, vm.asResource(), async () => {
+      await vm.create();
       await vm.navigateToDetail();
     });
   });
 
   it(
     'ID(CNV-2039) Creates windows 10 VM with correct metadata',
     async () => {
-      const testVMConfig = vmConfig(
-        'windows10',
-        testName,
-        provisionConfigs.get(ProvisionConfigName.CONTAINER),
-        _.cloneDeep(basicVMConfig),
-      );
-      testVMConfig.networkResources = [];
-      testVMConfig.operatingSystem = OperatingSystem.WINDOWS_10;
-      testVMConfig.flavorConfig.flavor = Flavor.MEDIUM;
-      testVMConfig.workloadProfile = WorkloadProfile.DESKTOP;
-      testVMConfig.startOnCreation = false; // do not check as there is only medium/large profile present and we would get insufficient memory.
-      const osID = OSIDLookup[testVMConfig.operatingSystem];
-
-      const vm = await wizard.createVirtualMachine(testVMConfig);
+      const builder = new VMBuilder()
+        .setNamespace(testName)
+        .setProvisionSource(provisionSources.Container)
+        .setOS(OperatingSystem.WINDOWS_10)
+        .setFlavor(flavorConfigs.Medium)
+        .setWorkload(Workload.DESKTOP)
+        .setDisks([hddDisk])
+        .setStartOnCreation(false)
```

Comment:
false by default?

Reply:
Yes, we didn't start this VM before either.

Reply:
I meant doesn't the builder output false value by default?

Reply:
Aah, yes sure

Reply:
\+ in multiple places this is not necessary